src/index.css

```diff
@@ -17,3 +16,18 @@ code {
 a {
   text-decoration: none;
 }
+
+.MuiMenuItem-root {
+  padding: 0 !important;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.MuiList-padding {
+  padding: 0 !important;
+}
+.MuiMenuItem-root {
```

Comment:
.MuiMenuItem-root is already written at line 20

Reply:
fixed